packaging/wildflydom4jjavaconf/java.conf

```diff
@@ -1 +1 @@
-JAVA_LIBDIR=/usr/share/ovirt-engine-wildfly/modules/system/layers/base/org:/usr/share/ovirt-engine-wildfly/modules/system/layers/base/org/apache/commons/collections/main:/opt/rh/eap7/root/usr/share/java
+JAVA_LIBDIR=/usr/share/ovirt-engine-wildfly/modules:/usr/share/ovirt-engine-extension-aaa-ldap/packaging/modules/org/ovirt/engine/extension/aaa/ldap/main/module.xml:/opt/rh/eap7/root/usr/share/java:/opt/rh/eap7/root/usr/share/java/wildfly/modules
```

Comment:
It won't work like this because here we need a PATH like JAVA_LIBDIR variable that will eventually be translated into java classpath  (-cp parameter). Every entry in  JAVA_LIBDIR must point to a directory which contains required jars (not xml or any 'parent' directories)
I doubt that for example module.xml  on class path would have any effect at all (unless something is reffering to it)